frontend/device/android/dictionaries.lua

```diff
@@ -0,0 +1,56 @@
+return { --[[ supported android dictionary applications.
+
+Most of them should support Intent.ACTION_SEND or Intent.ACTION_SEARCH.
+
+Some applications implement their custom intents. Colordict, for example,
+uses colordict.intent.action.SEARCH. ]]--
+
+    { "Aard2", "Aard2", false,
+        {
+            package="itkach.aard2",
+            action="aard2",
```

Comment:
If there is never anything else than these 2 "package" and "action", you could as well have them in the main list, and access them with v[4] and v[5].

```
{ "Aard2", "Aard2", false, "itkach.aard2", "aard2"},
{ "Alpus", "Alpus", false, "com.ngcomputing.fora.android", "search"}
{ "ColorDict", "ColorDict", false, "com.socialnmobile.colordict", "colordict"}
[...]
```
would look nice enough.